review.md

```diff
@@ -0,0 +1,422 @@
+I believe you would like ideas about how to optimize some functions. Please let me konw which functions you mean and share with me a reproducible example (with the package <https://reprex.tidyverse.org/>) showing me how to use the function I need to inspect. Here I write what I see as I review the structure of the package. For a more complete set of recomendations see <https://devguide.ropensci.org/guide-for-authors.html> and <http://r-pkgs.had.co.nz/>.
```

Comment:
Here are my comments. They explain in more detail what I comment on the tab "Files changed" of the pull request.